app/controllers/projects_controller.rb

```diff
@@ -90,6 +90,24 @@ def show
 
     @project_eligible_to_edit = true if project.status == Project::APPROVED_STATUS && eligible_editor?
 
+    @project_metadata = @project.metadata
+    @project_id = @project_metadata[:project_id]
```

Comment:
Did you mean to default to an empty hash so the rest of the calls would not fail if the metadata was nil?
```suggestion
    @project_id = @project_metadata[:project_id] || { }
```